core/services/workflows/engine.go

```diff
@@ -1172,6 +1204,8 @@ func NewEngine(cfg Config) (engine *Engine, err error) {
 
 	workflow, err := Parse(cfg.Workflow)
 	if err != nil {
+		cma := custmsg.NewLabeler().With(wIDKey, cfg.WorkflowID, woIDKey, cfg.WorkflowOwner, wnKey, cfg.WorkflowName)
```

Comment:
This is duplicating the creation of the labeler on line 1218: can we move this slightly higher up and reuse the labeler in both places :) ?